providers/org/arxiv/normalizer.py

```diff
@@ -0,0 +1,18 @@
+from share.normalize import *  # noqa
+
+
+class Person(Parser):
```

Comment:
Not currently, if you come up with a way to express it I'll show you how to extend the parser system.